pkg/cmd/pulumi/new_test.go

```diff
@@ -296,6 +299,46 @@ func TestGeneratingProjectWithExistingPromptedNameSucceeds(t *testing.T) {
 	assert.Equal(t, projectName, proj.Name.String())
 }
 
+//nolint:paralleltest // changes directory for process
+func TestCreatingProjectWithEmptyConfig(t *testing.T) {
+	// Regression test for https://github.com/pulumi/pulumi/issues/4081
+	skipIfShortOrNoPulumiAccessToken(t)
+
+	tempdir, _ := os.MkdirTemp("", "test-env")
+	defer os.RemoveAll(tempdir)
```

Comment:
Nit: I realize this is just following what other tests in this file do, but we could move to using the built-in `TempDir()` (when we're not keeping around the temp dir during failure):

```suggestion
	tempdir := t.TempDir()
```